src/Coin.cpp

```diff
@@ -60,6 +60,7 @@ using namespace std;
 
 // Map with coin entry dispatchers, key is coin type
 map<TWCoinType, CoinEntry*> dispatchMap = {}; 
+uint8_t dispatchMapInitialized = 0; // 0 not, 1 initialized, 2 in progress
```

Comment:
will std::atomic help? Probably we can define 1, 2 as an enum?

Reply:
I changed into 2 bool variables